src/app/app.css

```diff
@@ -156,6 +156,21 @@ html[data-useragent*='Linux'] {
       padding: 0;
     }
   }
+
+  :global(.expanded) {
+    & + .bodyWrapper {
+      padding-left: var(--side-bar-shrunk-size);
+    }
+  }
+
+  html[data-useragent*='Windows'],
+  html[data-useragent*='Linux'] {
+    & :global(.expanded) {
+      & + .bodyWrapper {
+        padding-left: calc(var(--side-bar-shrunk-size));
+      }
+    }
+  }
```

Comment:
Isn't the global.css file a better place for these styles?

Reply:
I don't think so, because this is just affecting to the `bodyWrapper` at `app.js`